### ISSUE TITLE
Topology.from_openmm() doesn't keep residue ids

### ISSUE THREAD

**Issue by dprada** [mdtraj/mdtraj#1427]
The method mdtraj.Topology.from_openmm() does not keep the residue ids. Is this a bug? 
Could this method have an argument similar to 'keepIDs' from  [simtk.openmm.app.pdbfile.PDBFile](http://docs.openmm.org/development/api-python/generated/simtk.openmm.app.pdbfile.PDBFile.html)?

Here an example of the reset on the residue indices imported from the openmm topology:

```python
# Saving a pdb file with mdtraj
system_mdtraj = md.load_pdb('https://files.rcsb.org/download/4ZP4.pdb')
system_mdtraj.save('4ZP4.pdb')

# First residue index with mdtraj
system_mdtraj = md.load_pdb('4ZP4.pdb')
print('With mdtraj:',system_mdtraj.topology.residue(0))

# First residue index with openmm
system_openmm = PDBFile('4ZP4.pdb')
residue_0 = next(system_openmm.topology.residues())
print('With OpenMM:',residue_0.id)

# First residue index with pdbfixer
system_pdbfixer = PDBFixer('4ZP4.pdb')
residue_0 = next(system_pdbfixer.topology.residues())
print('With PDBFixer:',residue_0.id)

# First residue index with Trajectory.from_openmm()
topology_mdtraj = mdtraj.Topology.from_openmm(system_openmm.topology)
print('With mdtraj.Topology.from_openmm:',topology_mdtraj.residue(0))
```

```bash
With mdtraj: GLU98
With OpenMM: 98
With PDBFixer: 98
With mdtraj.Topology.from_openmm: GLU0
```

**Comment by JBGreisman** [comment]
This issue was fixed in #1424, which was incorporated in mdtraj 1.9.3 release. Please update if you still run into issues with cross-compatibility of residue numbering with the new version.

**Comment by dprada** [comment]
Thanks! I will update.
